.github/workflows/review.yaml

```
@@ -0,0 +1,87 @@
+name: Release 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)